### ISSUE TITLE
Content selection checkboxes do not show content in updated version

### ISSUE THREAD

**Issue by ivanistheone** [learningequality/kolibri#6521]
### Observed behavior

Using the IMPORT MORE function in Kolibri shows two videos in the testing folder:
![Screen Shot 2020-02-05 at 1 06 15 PM](https://user-images.githubusercontent.com/163966/73869511-5b83ca00-4818-11ea-8463-65af90721070.png)

### Expected behavior
There should be three videos in that folder, two alrady on device and one new 
![Screen Shot 2020-02-05 at 1 06 15 PM](https://user-images.githubusercontent.com/163966/73869704-ce8d4080-4818-11ea-8ed0-8b2dd654efb1.png)

see [on studio](https://studio.learningequality.org/channels/2d9ab082d771408eb7b48cd7d888e2e6/edit/4c98f9d).

The [db file](http://studio.learningequality.org/content/databases/2d9ab082d771408eb7b48cd7d888e2e6.sqlite3) served by Studio is v2 and it contains the new video.

Kolibri also seems to have read the correct DB file during UPDATE CHANNEL wizard, because the UI shows there are 7 content items in the channel and 6 already on device:
![Screen Shot 2020-02-05 at 1 11 48 PM](https://user-images.githubusercontent.com/163966/73869878-17dd9000-4819-11ea-8d50-4208575bcee8.png)
so problem seems to be just related to the selection component.

### User-facing consequences
Users will not be able to import new content added to their channel.

### Errors and logs

No errors in the logs, just standard import logging.

#### During initial import of v1

```
INFO 2020-02-05 17:46:27,191 importchannel Downloading data for channel id 2d9ab082d771408eb7b48cd7d888e2e6
INFO 2020-02-05 17:46:27,892 channel_import Importing ContentTag data
INFO 2020-02-05 17:46:27,894 channel_import Importing ContentNode_has_prerequisite data
INFO 2020-02-05 17:46:27,895 channel_import Importing ContentNode_related data
INFO 2020-02-05 17:46:27,895 channel_import Importing ContentNode_tags data
INFO 2020-02-05 17:46:27,896 channel_import Importing ContentNode data
INFO 2020-02-05 17:46:27,898 channel_import Importing Language data
INFO 2020-02-05 17:46:27,898 channel_import Importing File data
INFO 2020-02-05 17:46:27,899 channel_import Importing LocalFile data
INFO 2020-02-05 17:46:27,900 channel_import Importing AssessmentMetaData data
INFO 2020-02-05 17:46:27,901 channel_import Importing ChannelMetadata data
INFO 2020-02-05 17:46:27,919 channel_import Channel 2d9ab082d771408eb7b48cd7d888e2e6 successfully imported into the database
INFO 2020-02-05 17:46:43,788 annotation Setting availability to True of 10 LocalFile objects based on passed in checksums
INFO 2020-02-05 17:46:43,802 annotation Setting availability of non-topic ContentNode objects based on LocalFile availability
INFO 2020-02-05 17:46:43,862 annotation Annotating ContentNode objects with children for 2 levels
INFO 2020-02-05 17:46:43,868 annotation Annotating ContentNode objects with children for level 2
INFO 2020-02-05 17:46:43,870 annotation Annotating ContentNode objects with children for level 1
```

After clicking VIEW CHANGES to preview what has changed in v1 --> v2:
```
INFO 2020-02-05 17:48:06,690 importchannel Downloading data for channel id 2d9ab082d771408eb7b48cd7d888e2e6
INFO 2020-02-05 17:48:07,320 channel_import Importing ContentTag data
INFO 2020-02-05 17:48:07,321 channel_import Importing ContentNode_has_prerequisite data
INFO 2020-02-05 17:48:07,321 channel_import Importing ContentNode_related data
INFO 2020-02-05 17:48:07,322 channel_import Importing ContentNode_tags data
INFO 2020-02-05 17:48:07,323 channel_import Importing ContentNode data
INFO 2020-02-05 17:48:07,323 channel_import Importing Language data
INFO 2020-02-05 17:48:07,324 channel_import Importing File data
INFO 2020-02-05 17:48:07,325 channel_import Importing LocalFile data
INFO 2020-02-05 17:48:07,326 channel_import Importing AssessmentMetaData data
INFO 2020-02-05 17:48:07,326 channel_import Importing ChannelMetadata data
INFO 2020-02-05 17:48:07,348 annotation Setting availability of LocalFile objects based on disk availability
INFO 2020-02-05 17:48:07,422 annotation Setting availability to True of 10 LocalFile objects based on passed in checksums
```

During the UPDATE CHANNEL:
```
INFO 2020-02-05 17:48:21,811 importchannel Downloading data for channel id 2d9ab082d771408eb7b48cd7d888e2e6
INFO 2020-02-05 17:48:21,867 channel_import Older version 1 of channel 2d9ab082d771408eb7b48cd7d888e2e6 already exists in database; removing old entries so we can upgrade to version 2
INFO 2020-02-05 17:48:21,887 channel_import Importing ContentTag data
INFO 2020-02-05 17:48:21,888 channel_import Importing ContentNode_has_prerequisite data
INFO 2020-02-05 17:48:21,889 channel_import Importing ContentNode_related data
INFO 2020-02-05 17:48:21,890 channel_import Importing ContentNode_tags data
INFO 2020-02-05 17:48:21,890 channel_import Importing ContentNode data
INFO 2020-02-05 17:48:21,892 channel_import Importing Language data
INFO 2020-02-05 17:48:21,892 channel_import Importing File data
INFO 2020-02-05 17:48:21,893 channel_import Importing LocalFile data
INFO 2020-02-05 17:48:21,894 channel_import Importing AssessmentMetaData data
INFO 2020-02-05 17:48:21,894 channel_import Importing ChannelMetadata data
INFO 2020-02-05 17:48:21,909 channel_import Channel 2d9ab082d771408eb7b48cd7d888e2e6 successfully imported into the database
INFO 2020-02-05 17:48:21,915 annotation Setting availability of non-topic ContentNode objects based on LocalFile availability
INFO 2020-02-05 17:48:21,930 annotation Annotating ContentNode objects with children for 2 levels
INFO 2020-02-05 17:48:21,935 annotation Annotating ContentNode objects with children for level 2
INFO 2020-02-05 17:48:21,938 annotation Annotating ContentNode objects with children for level 1
```

…

### Steps to reproduce
 - Import channel v1 form Studio
 - add content to channel on Studio and publish v2
 - use UPDATE channel and IMPORT MORE functionality to try to get new content

…

### Context
 * Kolibri version: 0.13
 * Operating system: linux
 * Browser: chome

**Comment by ivanistheone** [comment]
@jonboiser said: The ‘coral reefs’ node, if you look at the contentnode_granular network call has a ‘importable: false’ flag, which is why it’s not being shown on the UI.

@rtibbles said: Ah, this is probably an issue with the importability annotation not being refreshed on version change then. Because they are expensive, they are cached for an hour.

**Comment by rtibbles** [comment]
@ivanistheone said: the simplest solution is to clear these caches after a channel is updated.

**Comment by laurenlichtman** [comment]
I shared a video with @lyw07 demonstrating what occurred when I did this with 0.13.1b1.

**Comment by jonboiser** [comment]
@ivanistheone Is there a workaround you know for this? If this is indeed a cache issue, then usually restarting the server would fix it.

**Comment by rtibbles** [comment]
Note that in this case the cache used for the import availability is persisted to disk for it to be shared across processes, so restarting will not clear this particular cache. Clearing it on channel upgrade seems like the simplest. Will do a quick PR for this tomorrow.

**Comment by rtibbles** [comment]
As a least resistance fix, I am only going to clear the cache for Studio annotation, as that is the most likely to change in this scenario - and also the simplest to clear.

**Comment by rtibbles** [comment]
Peer and disk import will rely on the cache timeout instead - which seems reasonable, as they will tend to update on much longer timescales.

**Comment by ivanistheone** [comment]
> Peer and disk import will rely on the cache timeout instead - which seems reasonable, as they will tend to update on much longer timescales.

That seems reasonable indeed.

The only scenario I can think of where this could be an issue is someone trying to do a device-to-device transfer of a large channel (e.g. 32G) using an external disk that doesn't fit the whole channel (e.g. 16G), using an approach like:
 - export chunk 1
 - import chunk 1
 - rm -rf on disk
 - export chunk 2
 - import chunk 2

It's a prettttty rare scenario though, and the action of writing 16G-worth-of-content to an sdcard will probably take > 1h so might not even occur.

**Comment by rtibbles** [comment]
I changed my mind and just implemented the full thing just for consistency.

**Comment by rtibbles** [comment]
See PR above!

**Comment by rtibbles** [comment]
Oh, actually read through your scenario - yeah, nothing here would fix that, because that wouldn't be counted as an upgrade, and we have no other way of knowing that any change has taken place.

**Comment by rtibbles** [comment]
Fixed in #6533
